apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -25,7 +25,7 @@ export default class VersionControl {
   public static getChangedFiles(prefix?: string, targetBranch?: string): string[] {
     const branchName: string = targetBranch ? targetBranch : 'origin/master';
     const output: string = child_process
-      .execSync(`git diff ${branchName}... --name-only --diff-filter=A`)
+      .execSync(`git diff ${branchName}... --name-only --no-renames --diff-filter=A`)
```

Comment:
>-no-renames [](start = 56, length = 11)

@lijunle what if someone renames the file AND edits it, in the same PR?

Reply:
Git will detect it as old file removed and new file added. So, this command will output it. Otherwise, this command will output nothing.